apps/lepton/lepton.yml

```diff
@@ -0,0 +1,14 @@
+name: Lepton
+description: 'Open-source GitHub Gist client built for macOS, Windows and Linux. '
```

Comment:
There's an extra space at the end of the description. Could probably live without the "for macOs... " part too.